TGC/localisation/00_modifiers.csv

```diff
@@ -1497,3 +1497,19 @@ research_campaign;Active Research Campaign;;;;;;;;;;;;;x
 research_campaign_desc;We are currently focusing part of our annual budget on funding more research for our country;;;;;;;;;;;;;x
 church_property_confiscation;Church Property Confiscated;;;;;;;;;;;;;x
 church_property_confiscation_desc;We are fixing our budget holes with the property of the Catholic Church and its monasteries;;;;;;;;;;;;;x
+Adopted_latin;Adopted Latin;;;;;;;;;;;;;x
+Adopted_latin_desc;Adopted the Latin alphabet;;;;;;;;;;;;;x
+Adopted_Cyrilic;Adopted Cyrilic;;;;;;;;;;;;;x
+Adopted_Cyrilic_desc;Adopted the Cyrilic alphabet;;;;;;;;;;;;;x
+Adopted_Hellenic;Adopted Hellenic;;;;;;;;;;;;;x
+Adopted_Hellenic_desc;Adopted the Hellenic alphabet;;;;;;;;;;;;;x
+Adopted_Arabic;Adopted Arabic;;;;;;;;;;;;;x
+Adopted_Arabic_desc;Adopted the Arabic system;;;;;;;;;;;;;x
+Adopted_Ethiopic;Adopted Ethiopic;;;;;;;;;;;;;x
```

Comment:
Ethiopic is not a word, should be Ethiopian

Reply:
Ethiopian Semitic (also Ethio-Semitic, Ethiosemitic, Ethiopic or Abyssinian[[1]](https://en.wikipedia.org/wiki/Ethiopian_Semitic_languages#cite_note-Z2-5-1)) i